### ISSUE TITLE
Server rejected event

### ISSUE THREAD

**Issue by slayer** [kubernetes-sigs/descheduler#229]
There is error message in log:

```
E0205 11:22:06.986200       1 event.go:263] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"xxxx-web-db6785fc9-v6f7q.15f07dbea482b25a", GenerateName:"", Namespace:"xxxx-production", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:v1.Time{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, DeletionTimestamp:(*v1.Time)(nil), DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Pod", Namespace:"xxxx-production", Name:"xxxx-web-db6785fc9-v6f7q", UID:"22ae8c9e-4809-11ea-b920-4201ac100108", APIVersion:"v1", ResourceVersion:"27132912", FieldPath:""}, Reason:"Descheduled", Message:"pod evicted by sigs.k8s.io/descheduler", Source:v1.EventSource{Component:"sigs.k8s.io.descheduler", Host:""}, FirstTimestamp:v1.Time{Time:time.Time{wall:0xbf86c737b97a865a, ext:158538930, loc:(*time.Location)(0x211e400)}}, LastTimestamp:v1.Time{Time:time.Time{wall:0xbf86c737b97a865a, ext:158538930, loc:(*time.Location)(0x211e400)}}, Count:1, Type:"Normal", EventTime:v1.MicroTime{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'events is forbidden: User "system:serviceaccount:kube-system:descheduler-sa" cannot create resource "events" in API group "" in the namespace "xxxx-production"' (will not retry!)                                                                     
```

Is that ok ?

**Comment by ingvagabund** [comment]
Hey @slayer, did you use https://github.com/kubernetes-sigs/descheduler/#create-a-cluster-role ?

**Comment by seanmalloy** [comment]
The important part of the error message is all the way at the end `User "system:serviceaccount:kube-system:descheduler-sa" cannot create resource "events" in API group "" in the namespace "xxxx-production"' (will not retry!)`. I added k8s Event support in #183, but I did not update the `ClusterRole` in k8s YAML files to handle this.

FYI ... on the master branch the `ClusterRole` is actually duplicated in multiple places right now.
* https://github.com/kubernetes-sigs/descheduler#create-a-cluster-role
* https://github.com/kubernetes-sigs/descheduler/blob/master/kubernetes/rbac.yaml#L2
https://github.com/kubernetes-sigs/descheduler/blob/master/examples/descheduler.yaml#L2

Pull request #208 will remove all of the above mentioned duplication.

**Comment by seanmalloy** [comment]
@ingvagabund I can fix the issue with the `ClusterRole` in my PR #208 if you want. Let me know if you want me to do this.

**Comment by ingvagabund** [comment]
@seanmalloy sounds good as long as it's in a separate PR. Thanks!!!

**Comment by seanmalloy** [comment]
/assign @seanmalloy 

I will fix this after #208 is merged.

**Comment by seanmalloy** [comment]
@slayer this is fixed in #237
